core/src/mill/define/BaseModule.scala

```diff
@@ -2,52 +2,53 @@ package mill.define
 
 import ammonite.ops.Path
 
-object BaseModule{
+object BaseModule {
   case class Implicit(value: BaseModule)
 }
 
 abstract class BaseModule(millSourcePath0: Path,
                           external0: Boolean = false,
-                          foreign0 : Boolean = false)
-                         (implicit millModuleEnclosing0: sourcecode.Enclosing,
-                          millModuleLine0: sourcecode.Line,
-                          millName0: sourcecode.Name,
-                          millFile0: sourcecode.File)
-  extends Module()(
-    mill.define.Ctx.make(
-      implicitly,
-      implicitly,
-      implicitly,
-      BasePath(millSourcePath0),
-      Segments(),
-      mill.util.Router.Overrides(0),
-      Ctx.External(external0),
-      Ctx.Foreign(foreign0),
-      millFile0
-    )
-  ){
+                          foreign0: Boolean = false)(
+    implicit millModuleEnclosing0: sourcecode.Enclosing,
+    millModuleLine0: sourcecode.Line,
+    millName0: sourcecode.Name,
+    millFile0: sourcecode.File)
+    extends Module()(
+      mill.define.Ctx.make(
+        implicitly,
+        implicitly,
+        implicitly,
+        BasePath(millSourcePath0),
+        Segments(),
+        mill.util.Router.Overrides(0),
+        Ctx.External(external0),
+        Ctx.Foreign(foreign0),
+        millFile0
+      )
+    ) {
   // A BaseModule should provide an empty Segments list to it's children, since
   // it is the root of the module tree, and thus must not include it's own
   // sourcecode.Name as part of the list,
   override implicit def millModuleSegments: Segments = Segments()
   override def millSourcePath = millOuterCtx.millSourcePath
   override implicit def millModuleBasePath: BasePath = BasePath(millSourcePath)
-  implicit def millImplicitBaseModule: BaseModule.Implicit = BaseModule.Implicit(this)
+  implicit def millImplicitBaseModule: BaseModule.Implicit =
+    BaseModule.Implicit(this)
   def millDiscover: Discover[this.type]
 }
 
-
-abstract class ExternalModule(implicit millModuleEnclosing0: sourcecode.Enclosing,
-                              millModuleLine0: sourcecode.Line,
-                              millName0: sourcecode.Name)
-  extends BaseModule(ammonite.ops.pwd, external0 = true, foreign0 = false){
+abstract class ExternalModule(
+    implicit millModuleEnclosing0: sourcecode.Enclosing,
+    millModuleLine0: sourcecode.Line,
+    millName0: sourcecode.Name)
```

Comment:
In general I would like definitions to never include a newline after the opening paren. I especially don't like the fact that after this reformatting, sometimes there's a newline and sometimes there isn't...